backend/sftp/sftp.go

```diff
@@ -903,6 +944,34 @@ func parseHash(bytes []byte) string {
 	return strings.Split(string(bytes), " ")[0] // Split at hash / filename separator
 }
 
+// Parses the byte array output from the SSH session
+// returned by an invocation of df into
+// the disk size, used space, and avaliable space on the disk, in that order.
+// Only works when `df` has output info on only one disk
+func parseUsage(bytes []byte) (int64, int64, int64) {
```

Comment:
I'd really like to see a test for parseUsage.  I'd write a table driven test like the ones here: https://github.com/ncw/rclone/blob/master/backend/sftp/sftp_internal_test.go

We can then get the output of `df -k` from various OSes as test cases.

Reply:
Okay, I put 2 test cases in.